admin/upload-rpms

```diff
@@ -1,125 +1,13 @@
 #!/usr/bin/env python
```

Comment:
:question: I'm tempted to suggest this should actually be called `publish-rpms`.